critiquebrainz/frontend/translations/fr/LC_MESSAGES/messages.po

```diff
@@ -413,20 +414,20 @@ msgstr "Emplacement"
 #: critiquebrainz/frontend/templates/macros.html:53
 #: critiquebrainz/frontend/templates/search/selector.html:10
 msgid "Release Group"
-msgstr ""
+msgstr "Groupe de parution"
 
 #: critiquebrainz/frontend/templates/macros.html:69
 #, python-format
 msgid "Review by %(user)s"
-msgstr "Critique par %(user)s"
+msgstr "Critique de %(user)s"
 
 #: critiquebrainz/frontend/templates/macros.html:79
 msgid "View Spotify mapping information"
 msgstr ""
 
 #: critiquebrainz/frontend/templates/macros.html:85
 msgid ""
-"We don't have a mapping between this release group and Spotify. Please help "
+"We don’t have a mapping between this release group and Spotify. Please help "
```

Comment:
It looks like you updated the quote in the msgid here instead of editing the msgstr

Reply:
Fixed by https://github.com/metabrainz/critiquebrainz/pull/299/commits/2078501b2bbc994b58aaf87892080aee2e33ad90